app/components/recirculation-prefooter.js

```diff
@@ -24,6 +24,40 @@ export default Component.extend(
 		isVisible: false,
 		liftigniter: inject.service(),
 		i18n: inject.service(),
+		hasNoLiftigniterSponsoredItem: true,
```

Comment:
I would reduce it here over items here not to couple this with didEnterViewport and side effect of some other maps